liberty-maven-plugin/src/it/server-config-props-it/cleanTest.bsh

```diff
@@ -0,0 +1,4 @@
+//Runs clean in the test project directory
+Runtime.getRuntime().exec("../../../../mvnw clean", null, basedir);
```

Comment:
Why did you add this file here? It exists in the src/test/resources directory and is run based off the pom.xml configuration `<postBuildHookScript>cleanTest</postBuildHookScript>`. It has been working up until now. Did you run into a problem?

Reply:
this was a mistake - I removed